package.json

```diff
@@ -88,5 +90,16 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
```

Comment:
So if this gets run before commit. It runs `npm run lint:fix`, however, does it actually stage those files? Or would it run the lint, fix the files, commit, then still have the lint changes in the working copy?

Reply:
`lint-staged` is a library that runs the specified command (in this case linting) only on staged files, therefore, your PR will have only stuff that you changed (because if linter would run globally and somebody would push without checks, you might style fix his file and it would end up in your PR).

`lint-staged` will also stage files that got fixed by linter that was run by husky. Maybe that's why you didn't like husky because it was used without lint-staged.